test/ir/pir/translator/CMakeLists.txt

```
@@ -16,6 +16,9 @@ list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST
 list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_nop_translator)
 list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_partial_send_translator)
 list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_partial_recv_translator)
+list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_c_reduce_max_translator)
+list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_c_reduce_prod_translator)
+list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_c_scatter_translator)
```

```suggestion
list(APPEND DISTRIBUTED_OP_TRANSLATOR_TEST test_c_scatter_translator)
```
这里合并冲突的时候有点问题，另外test_c_scatter_translator按字母序排列，后续PR合入会减少冲突

---

收到